include/cantera/kinetics/Kinetics.h

```diff
@@ -1009,6 +1279,9 @@ class Kinetics
     //! Forward rate constant for each reaction
     vector_fp m_rfn;
 
+    //! Delta G^0 times gas constant and temperature for all reactions
+    vector_fp m_delta_gibbs0_RT;
```

Comment:
I am fairly certain that this docstring is correct - there are some instances where comments erroneously refer to this quantity as `Delta G^0`.

Reply:
Right, in the expression `exp(m_rkcn[irxn]*rrt - m_dn[irxn]*m_logStandConc)`, it's dividing `Delta G^0` by `RT` to get a nondimensional quantity, since the argument to `exp()` should be nondimensional.

Reply:
Makes sense. I guess there is a discrepancy in nomenclature.

Reply:
Fixed.